src/main/java/seedu/address/model/transaction/TransactionContainsKeywordsAndPersonNamesPredicate.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.transaction;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.person.Name;
+
+/**
+ * Tests that a {@code Transaction} contains any of the keywords and any of the names of all the {@code Person},
+ * either as a payer or a payee. Name matching is exact and case-insensitive.
```

Comment:
```suggestion
 * either as a payer or a payee. Name matching is exact and case-sensitive.
```
Isn't it still case-sensitive since it is an exact matching?